src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseStatistics.cpp

```diff
@@ -176,21 +206,32 @@ void SinglePhaseStatistics::computeRegionStatistics( MeshLevel & mesh,
     RegionStatistics & regionStatistics = region.getReference< RegionStatistics >( viewKeyStruct::regionStatisticsString() );
 
     regionStatistics.minPressure = MpiWrapper::min( regionStatistics.minPressure );
+    regionStatistics.averagePressure = MpiWrapper::sum( regionStatistics.averagePressure );
     regionStatistics.maxPressure = MpiWrapper::max( regionStatistics.maxPressure );
+
     regionStatistics.minDeltaPressure = MpiWrapper::min( regionStatistics.minDeltaPressure );
     regionStatistics.maxDeltaPressure = MpiWrapper::max( regionStatistics.maxDeltaPressure );
+
+    regionStatistics.minTemperature = MpiWrapper::min( regionStatistics.minTemperature );
+    regionStatistics.averageTemperature = MpiWrapper::sum( regionStatistics.averageTemperature );
+    regionStatistics.maxTemperature = MpiWrapper::max( regionStatistics.maxTemperature );
+
     regionStatistics.totalUncompactedPoreVolume = MpiWrapper::sum( regionStatistics.totalUncompactedPoreVolume );
     regionStatistics.totalPoreVolume = MpiWrapper::sum( regionStatistics.totalPoreVolume );
-    regionStatistics.averagePressure = MpiWrapper::sum( regionStatistics.averagePressure );
+    regionStatistics.totalMass = MpiWrapper::sum( regionStatistics.totalMass );
+
     if( regionStatistics.totalUncompactedPoreVolume > 0 )
     {
-      regionStatistics.averagePressure /= regionStatistics.totalUncompactedPoreVolume;
+      float invTotalUncompactedPoreVolume = 1.0 / regionStatistics.totalUncompactedPoreVolume;
+      regionStatistics.averagePressure *= invTotalUncompactedPoreVolume;
+      regionStatistics.averageTemperature *= invTotalUncompactedPoreVolume;
     }
     else
     {
       regionStatistics.averagePressure = 0.0;
+      regionStatistics.averageTemperature = 0.0;
       GEOS_LOG_LEVEL_RANK_0( 1, getName() << ", " << regionNames[i]
```

Comment:
should it be warning?

Reply:
Indeed, as those values can be silently output in a `.csv`, we should output that as a warning.